### DISCUSSION TITLE
Wizzard: share data between steps with repeater

### DISCUSSION THREAD

**Discussion by photojo** [filamentphp/filament#13162] [Help]
### Package

Form builder

### Package Version

2.0

### How can we help you?

Why I can't get data from previous step in a ?

F.e. in step 5 I want do access value from step 3.

Step 3
`... Select::make('product_id') ...`

Step 5
```
Repeater::make('content')->schema([
Radio::make('value')
->options(fn(Closure $get) =>Product::where('product_id',$get('product_id'))->get()->pluck('selling')->toArray()
])

```
I also tried where('product_id',$get('../../product_id'))

I do not get an error, but also no value from step 3.

What's wrong?

**Comment by leandrocfe** [comment]
It was supposed to work with `$get('../../product_id')`
Radio options should return an array with key/value
